veloroutes_voies_vertes/install/sql/export/export_poi_portion.sql

```diff
@@ -0,0 +1,20 @@
+-- Select les poi à moins de 100m d'une portion cyclable
+SELECT
+	poi.id_poi AS ID_POI,
+	vp.id_portion AS ID_PORTION
+FROM veloroutes.v_portion as vp, veloroutes.poi_service as poi
+WHERE ST_Distance(poi.geom, vp.geom)< 100;
+
+-- Select FROM poi_acces
+SELECT
+	poi.id_poi AS ID_POI,
+	vp.id_portion AS ID_PORTION
+FROM veloroutes.v_portion as vp, veloroutes.poi_acces as poi
+WHERE ST_Distance(poi.geom, vp.geom)< 100;
+
+-- Select FROM poi_tourisme
+SELECT
+	poi.id_poi AS ID_POI,
+	vp.id_portion AS ID_PORTION
+FROM veloroutes.v_portion as vp, veloroutes.poi_tourisme as poi
+WHERE ST_Distance(poi.geom, vp.geom)< 100;
```

Comment:
Ce fichier contient 3 requêtes distinctes. Il ne peut donc aps être utilisé tel quel pour créer une couche

Reply:
Work in progress